DataFormats/interface/Electron.h

```diff
@@ -33,6 +33,8 @@ namespace cat {
       if( dR < 0.35) return relIso03_;
       else return relIso04_;
     }
+
+    float MiniRelIso() const {return relMiniIso03_;}
```

Comment:
I have moved this to Lepton.h and changed the name. I will make new pull request.